Forms/main.css

```diff
@@ -1,57 +1,84 @@
 body{
-    width: 1800px;
-    height: 900px;
-    /*background-color: yellow;*/
+    width: 100vw;
+    height: 100vh;
 }
 
 div.container{
     display: flex;
-    margin: 10px    ;
+    margin: 10px;
+    justify-content: center;
+}
+div.head-container{
+    display: flex;
+    margin: 10px;
+    flex-direction: row-reverse;
+    justify-content: center;
 }
-
 h1{
     color: darkgray;
-    width: 95%;
+    width: 100%;
     height: 100px;
     text-align: center;
+    padding-right: 200px;
 }
 
 footer{
-    position: relative;
-    width: 95%;
-    height: 100px;
+    background-color: aquamarine;
+    width: 100% ;
+    height: 50px;
     text-align: center;
     margin: 40px 0 0 40px;
 }
 
 .img1 {
-    position: absolute;
+    position: relative;
+    display: flex;
     width: 200px;
     height: 150px;
-    margin: 10px;
-
+    margin-left: 40px;
 }
 
-div.first_collum{
-
+div.first_column{
+    width: 200px;
+    /*margin-left: 60px;*/
+    margin-right: 60px;
     display: flex;
```

Comment:
The order of css styles declaration should be always the same. I advise you to [look it up here](https://9elements.com/css-rule-order/#:~:text=So%20the%20order%20is%3A%20position,text%2D*%20%2C%20then%20word%2D*%20.) and stick to it.

Reply:
so in this exact case it would be better to rearrange the styles as follows:
display
justify-content
margin
width